### ISSUE TITLE
KeyError: 'frames'

### ISSUE THREAD

**Issue by judemont** [Gertje823/prezi2pdf#6]
Hi,
When I run the script I get this error:
```
Traceback (most recent call last):
  File "/home/judemont/autre/prezi2pdf/prezi2pdf.py", line 70, in <module>
    download_presentation(id)
  File "/home/judemont/autre/prezi2pdf/prezi2pdf.py", line 47, in download_presentation
    total = len(data['frames'])
                ~~~~^^^^^^^^^^
KeyError: 'frames'

```
The api on which the script is trying to access the frames (https://prezi.com/api/v2/storyboard/frames/d12zlicnmfl7/) returns the error message `The endpoint has been deprecated.`

**Comment by Gertje823** [comment]
Thanks for opening this issue. Unfortunately the endpoint is no longer supported. I fixed it in ba90577. Now the images are scraped from the storyboard API endpoint. The disadvantage is that the images are now only available in low quality.

**Comment by judemont** [comment]
It works, thank you 👍
